api/src/main/java/io/camunda/zeebe/process/test/api/ZeebeTestEngine.java

```diff
@@ -87,4 +88,9 @@ public interface ZeebeTestEngine {
    * @throws TimeoutException if the engine has not reached a busy state before the timeout
    */
   void waitForBusyState(Duration timeout) throws InterruptedException, TimeoutException;
+
+  /**
+   * @return a newly created {@link ZeebeClient} with custom mapper
+   */
+  ZeebeClient createClientWithCustomMapper(ObjectMapper customObjectMapper);
```

Comment:
❌ Let's overload the existing method and put it under the existing method in this interface for easier readability.